sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/healthcare/FhirIO.java

```diff
@@ -170,6 +184,21 @@
  * // Alternatively you could use import for high throughput to a new store.
  * FhirIO.Write.Result writeResult =
  *     output.apply("Import FHIR Resources", FhirIO.executeBundles(options.getNewFhirStore()));
+ *
+ * // Export FHIR resources to Google Cloud Storage.
+ * String fhirStoreName = ...;
+ * String exportGcsUriPrefix = ...;
+ * FhirIO.ExportGcs.Result exportResult =
```

Comment:
Should we return errors with exporting in the `Result`?

Reply:
Export extremely unlikely to have error with individual resources (and if there is it usually occurs for all resources). The error is usually at the LRO level, which the workflow will fail with IOException.

Reply:
In that case, you probably won't need to wrap it in a `Result` class, since you can return a `PCollection` directly.

Reply:
Done.